### PR TITLE
Fixed the console

### PR DESCRIPTION
**Pull request by JacksonSchoe**
Taylor please look into my implementation of hiding of the console and production, and look into testing it if possible because I wasn't sure how to test if it was hidden in production.

### CODE REVIEW

**Comment by JacksonSchoe** [comment]
image

File: src/main/js/App.js [hunk 1]
```diff
@@ -542,9 +542,19 @@ const App = () => {
                     </div>
                 </div>
                 <div>
-                    <div>{"Console"}</div>
-                    <textarea readOnly value={consoleText} class="console"></textarea>
-                </div>
+                    {process.env.NODE_ENV !== 'production' &&
```
Comment (TaylorAbraham) [inline src/main/js/App.js:545]: Good use of &&. Once we merge this, we should make sure to check that this does NOT appear on heroku.

File: src/main/js/App.js [hunk 2]
```diff
@@ -542,9 +542,19 @@ const App = () => {
                     </div>
                 </div>
                 <div>
-                    <div>{"Console"}</div>
-                    <textarea readOnly value={consoleText} class="console"></textarea>
```
Comment (TaylorAbraham) [inline src/main/js/App.js:546]: This used to have a class, we should add `className="console"` to the redone component.

**Review by TaylorAbraham — CHANGES_REQUESTED** [review]

**Review by TaylorAbraham — APPROVED** [review]
You fixed my comment, so looks all good now!

**Pull request merged**